pd-microservices-deployment-topology.md

```diff
@@ -0,0 +1,105 @@
+---
+title: PD 微服务部署拓扑
+summary: 了解在部署最小拓扑集群的基础上，部署 PD 微服务的拓扑结构。
+---
+
+# PD 微服务部署拓扑
+
+本文介绍在部署最小拓扑集群的基础上，部署 [PD 微服务](/pd-microservices.md) 的拓扑结构。
+
+## 拓扑信息
+
+| 实例                 | 个数 | 物理机配置                        | IP                                        | 配置                        |
+| :------------------- | :--- | :-------------------------------- | :---------------------------------------- | :-------------------------- |
+| TiDB                 | 2    | 16 VCore 32GB \* 1                | 10.0.1.1 <br/> 10.0.1.2                   | 默认端口 <br/> 全局目录配置 |
+| PD                   | 3    | 4 VCore 8GB \* 1                  | 10.0.1.3 <br/> 10.0.1.4 <br/> 10.0.1.5    | 默认端口 <br/> 全局目录配置 |
+| TSO                  | 2    | 4 VCore 8GB \* 1                  | 10.0.1.6 <br/> 10.0.1.7                   | 默认端口 <br/> 全局目录配置 |
+| Scheduling           | 2    | 4 VCore 8GB \* 1                  | 10.0.1.8 <br/> 10.0.1.9                   | 默认端口 <br/> 全局目录配置 |
+| TiKV                 | 3    | 16 VCore 32GB 2TB (nvme ssd) \* 1 | 10.0.1.10 <br/> 10.0.1.11 <br/> 10.0.1.12 | 默认端口 <br/> 全局目录配置 |
+| Monitoring & Grafana | 1    | 4 VCore 8GB \* 1 500GB (ssd)      | 10.0.1.13                                 | 默认端口 <br/> 全局目录配置 |
+
+### 拓扑模版
+
+<details>
+<summary>简单 PD 微服务配置模版</summary>
+
+```yaml
+# # Global variables are applied to all deployments and used as the default value of
+# # the deployments if a specific deployment value is missing.
+global:
+  user: "tidb"
+  ssh_port: 22
+  deploy_dir: "/tidb-deploy"
+  data_dir: "/tidb-data"
+  listen_host: 0.0.0.0
+  arch: "amd64"
+  pd_mode: "ms"
+
+monitored:
+  # # The communication port for reporting system information of each node in the TiDB cluster.
+  node_exporter_port: 9200
+  # # Blackbox_exporter communication port, used for TiDB cluster port monitoring.
+  blackbox_exporter_port: 9215
+  # # Storage directory for deployment files, startup scripts, and configuration files of monitoring components.
+  # deploy_dir: "/tidb-deploy/monitored-9100"
+  # # Data storage directory of monitoring components.
+  # data_dir: "/tidb-data/monitored-9100"
+  # # Log storage directory of the monitoring component.
+  # log_dir: "/tidb-deploy/monitored-9100/log"
```

Comment:
Not sure if the "monitored" configuration is necessary in this simple configuration template
```suggestion
```